src-app/app.tsx

```diff
@@ -1,11 +1,15 @@
-import { loadAppTheme } from '@app/core-modules/app-theme'
-import { StrictMode, Suspense } from 'react'
+import { useAppTheme } from '@app/core-modules/app-theme'
+import { StrictMode, Suspense, useEffect } from 'react'
 import { Router } from './routing'
 import './styles/index.css'
 
-loadAppTheme()
-
 const App = () => {
+	const { loadTheme } = useAppTheme()
+
+	useEffect(() => {
+		loadTheme()
```

Comment:
`loadTheme` can be used directly, without placing it, inside of `useEffect`.

Reply:
I only want it to be called once.